crds/templates/crds.yaml

```diff
@@ -1,3 +1,12 @@
+# To prevent accidental injection into istio-system.
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: {{ .Release.Namespace }}
```

Comment:
we may want to add all the namespaces here not just .Release.Namespace, can be in the future though

Reply:
That's a good idea - added.

Reply:
Please don't add namespaces to a file names 'crds...'

Reply:
Done.